assets/styles/desktop.css

```diff
@@ -0,0 +1,412 @@
+    width: 100%;
+    cursor: pointer;
+    transition: 0.5s ease-out;
+  }
+
+  .all-programs:hover {
+    border: 2px solid rgb(214, 71, 19);
+  }
+
+  .podium {
+    padding: 10px;
+    align-items: center;
+    background-image:
+      radial-gradient(
+        circle at 99% 33%,
+        hsl(235deg 0% 48% / 3%) 0%,
+        hsl(235deg 0% 48% / 3%) 37%,
+        transparent 37%,
+        transparent 100%
+      ),
+      radial-gradient(
+        circle at 46% 16%,
+        hsl(235deg 0% 48% / 3%) 0%,
+        hsl(235deg 0% 48% / 3%) 43%,
+        transparent 43%,
+        transparent 100%
+      ),
+      radial-gradient(
+        circle at 99% 25%,
+        hsl(235deg 0% 48% / 3%) 0%,
+        hsl(235deg 0% 48% / 3%) 22%,
+        transparent 22%,
+        transparent 100%
+      ),
+      radial-gradient(
+        circle at 57% 88%,
+        hsl(235deg 0% 48% / 3%) 0%,
+        hsl(235deg 0% 48% / 3%) 86%,
+        transparent 86%,
+        transparent 100%
+      ),
+      radial-gradient(
+        circle at 82% 78%,
+        hsl(235deg 0% 48% / 3%) 0%,
+        hsl(235deg 0% 48% / 3%) 22%,
+        transparent 22%,
+        transparent 100%
+      ),
+      linear-gradient(
+        90deg,
+        rgb(40 37 37),
+        rgb(40 37 37)
+      );
+    height: 100%;
+  }
+
+  .podium > h2 {
+    color: #ec5242;
+    font-size: 18px;
+    padding: 7px;
+  }
+
+  .podium > p {
+    padding: 10px 0;
+    font-size: 15px;
+    color: white;
+  }
+
+  .speakers-section {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    padding: 30px;
+  }
+
+  .speaker-container {
+    max-width: 500px;
+    height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .speaker-text {
+    width: 60%;
+    margin: 10px;
+  }
+
+  .para {
+    padding: 0 10px;
+    font-size: 0.8rem;
+    color: #ec5242;
+    font-style: italic;
+  }
+
+  .bio {
+    padding: 0 10px;
+    font-size: 1rem;
+  }
+
+  .name {
+    padding: 3px 10px;
+    font-size: 1.2rem;
+    font-weight: 800;
+  }
+
+  .speaker-heading {
+    width: 100%;
+    height: 100%;
+    text-align: center;
+    font-size: 1.5vw;
+    padding: 20px;
+    color: #272a31;
+  }
+
+  #see-program-link {
+    display: none;
+  }
+
+  .top-footer {
+    background-color: #272a31;
+    height: 300px;
+  }
+
+  .footer-h2 {
+    text-align: center;
+    color: #fff;
+  }
+
+  /* Clients Section */
+
+  #client-section::before {
+    content: "";
+    position: absolute;
+    width: 100%;
+    z-index: -1;
+    opacity: 1.9;
+  }
+
+  #clients {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .client-item {
+    padding: 24px;
+  }
+
+  #clients img {
+    height: 104px;
+  }
+
+  .bottom-footer {
+    background-color: black;
+    border: 1px solid black;
+    height: 200px;
+    margin-top: 20px;
+    color: #fff;
+    padding-top: 50px;
+    display: flex;
+  }
+
```

Comment:
- [ ] as requested by the previous reviewer https://github.com/Caren-Koroeny/hiddekel-global-summit/pull/1#discussion_r1061432102 please make sure the footer matches the design guidelines, how it is looking curently :point_down: 
![image](https://user-images.githubusercontent.com/97521557/210615278-178eb590-847a-4d0d-b0b6-d700a183dca9.png)

how it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/210615396-454f82d7-b424-4521-99f0-d1cdfcb08e58.png)

notice their background is different white background on home page and a dark gray background on the about page

Reply:
- @PinkMoon25  thank you I made the changes for both home and about page.

- ![Screenshot from 2023-01-04 21-48-35](https://user-images.githubusercontent.com/55842414/210627710-e50862ad-e226-4b5f-9302-c16e9313d994.png)

- 
![Screenshot from 2023-01-04 21-49-13](https://user-images.githubusercontent.com/55842414/210627880-a753c4d3-9511-4c88-a8ff-c1ab0ed2e4d5.png)